js_modules/dagit/packages/core/src/asset-graph/SidebarAssetInfo.tsx

```diff
@@ -56,12 +57,18 @@ export const SidebarAssetInfo: React.FC<{
   const {assetMetadata, assetType} = metadataForAssetNode(asset);
   const hasAssetMetadata = assetType || assetMetadata.length > 0;
   const assetConfigSchema = asset.configField?.configType;
+  const codeOriginInfo = asset.metadataEntries.find((e) => e.label === '__code_origin');
+
+  const codeOrigin =
+    codeOriginInfo &&
+    codeOriginInfo.__typename === 'JsonMetadataEntry' &&
+    JSON.parse(codeOriginInfo.jsonString);
```

Comment:
I wonder if we could export this as a helper from CodeLink.tsx as well. It seems like theres' a chance this could also end up on the Asset Details page, and it'd be nice to centralize the getter for the data.